.github/workflows/containerize-agent.yaml

```diff
@@ -117,13 +117,17 @@ jobs:
         id: get_version
         run: echo "POETRY_VERSION=$(poetry version -s)" >> $GITHUB_OUTPUT
 
+      - name: Get is_dev from pyproject.toml
+        id: get_is_dev
+        run: echo "IS_DEV=$(poetry version -s | grep -q 'dev' && echo true || echo false)" >> $GITHUB_OUTPUT
+
       - name: Get is_release from pyproject.toml
         id: get_is_release
-        run: echo "IS_RELEASE=$(poetry version -s | grep -q 'dev' && echo false || echo true)" >> $GITHUB_OUTPUT
+        run: echo "IS_RELEASE=$(poetry version -s | grep -q '[A-Za-z ]*' && echo false || echo true)" >> $GITHUB_OUTPUT
```

Comment:
You might also consider doing this with a python script/invoke task so that we are using the same method for determining versions.

See the `Version` class docs below 👇 

https://packaging.pypa.io/en/stable/version.html#packaging.version.Version